### PR TITLE
Enable CPM on windows

### PR DESCRIPTION
**Pull request by muodov**
**Asana Task/Github Issue:** https://app.asana.com/0/0/1205319036091446/1208775075004068/f

#### Reference

-   [Config Reviewer Documentation](https://app.asana.com/0/1200890834746050/1204443212791216/f)
-   [Config Maintainer Documentation](https://app.asana.com/0/1200890834746050/1200573250322769/f)
-   [Feature Implementer Documentation](https://app.asana.com/0/1200890834746050/1201498956177210/f)

### CODE REVIEW

**Review by szanto90balazs — APPROVED** [review]
Looks good, I force loaded with these settings and cookie management was still enabled prior to the upcoming code changes in https://github.com/duckduckgo/windows-browser/pull/2632/

File: overrides/windows-override.json [hunk 1]
```diff
@@ -77,12 +77,11 @@
             }
         },
         "autoconsent": {
-            "state": "disabled",
+            "state": "enabled",
             "minSupportedVersion": "0.50.0",
-            "settings": {
```
Comment (szanto90balazs) [inline overrides/windows-override.json:82]: You might want to keep the `settings` node, as the [AutoConsent.Service](https://github.com/duckduckgo/windows-browser/blob/657473e4a55ab81d019a7746d117ea1e1f01d53b/WindowsBrowser/Application/Autoconsent/Service.cs#L103) relies on its existence.

The updated config still seems have a `settings` node, though, with:

```
disabledCMPs ...
```

Comment (szanto90balazs) [reply overrides/windows-override.json:82]: As discussed in MM, I made manual edits on my local config. I didn't consider the base config, which I could have. Thanks for resolving this.

Comment (jonathanKingston) [reply overrides/windows-override.json:82]: > as the settings section, and this file is supposed to be applied on top of that

These won't merge. Is this your intention?

Comment (jonathanKingston) [reply overrides/windows-override.json:82]: Ignore the last comment, I was confused by the thread :)

**Pull request merged**